acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/bulkloading/AzureKeyVaultAcceptanceTest.java

```diff
@@ -60,12 +62,31 @@ void ensureSecretsInKeyVaultAreLoadedAndReportedViaPublicKeysApi() {
     final Response response = signer.callApiPublicKeys(KeyType.BLS);
     response.then().statusCode(200).contentType(ContentType.JSON).body("", contains(EXPECTED_KEY));
 
-    signer
-        .healthcheck()
+    // Since our Azure vault contains some invalid keys, the healthcheck would return 503.
+    final Response healthcheckResponse = signer.healthcheck();
+    healthcheckResponse
         .then()
-        .statusCode(200)
+        .statusCode(503)
```

Comment:
This is ok for now in fixing the build. But the expectation shouldn't be that we expect to fail loading keys i.e. have a 503 status code. Otherwise, we can't tell that having all valid keys works correctly. We could add a tag or use a prefix for the invalid key so they aren't used for this test.